src/plugins/home/public/application/components/tutorial/saved_objects_installer.js

```diff
@@ -62,9 +62,12 @@ class SavedObjectsInstallerUi extends React.Component {
 
     let resp;
     try {
-      resp = await this.props.bulkCreate(this.props.savedObjects, {
-        overwrite: this.state.overwrite,
-      });
+      resp = await this.props.bulkCreate(
+        // Filter out the saved object version field, if present, to avoid inadvertently triggering optimistic concurrency control.
+        // eslint-disable-next-line no-unused-vars
+        this.props.savedObjects.map(({ version, ...savedObject }) => savedObject),
```

Comment:
Nit: imo it'd be a little clearer what's going on here if this was pulled out into a variable instead of putting this all inside the function invocation